server/src/middleware.ts

```diff
@@ -53,6 +53,7 @@ export const userInfoSync = async (
         },
       });
     } catch (err) {
+      console.log(err);
```

Comment:
User logger.debug

In general always the logger. Produces the correct format in production